certchain/certmanager/augmentor.go

```diff
@@ -130,13 +127,18 @@ func (a *Augmentor) daemon(rcNext, orNext futureevent.Event) {
 				orNext, err = a.maintainOCSPResp()
 				if err != nil {
 					log.Printf("cannot update the OCSP response: %v", err)
+				} else {
```

Comment:
Note that I added these log statements to indicate that the OCSP retrieval was successful after several failed retries.